spyder/widgets/mixins.py

```diff
@@ -22,9 +22,11 @@
 from qtpy.QtCore import QPoint, Qt
 from qtpy.QtGui import QCursor, QTextCursor, QTextDocument
 from qtpy.QtWidgets import QApplication, QToolTip
-from qtpy import PYQT5
+from qtpy import PYQT_VERSION
 
-if PYQT5:
+PYQT55_VERSION = tuple(int(x) for x in PYQT_VERSION.split('.')) >= (5, 5)

```

Comment:
@rlaverde, although Qt and PyQt versions match (e.g. Qt 5.5 and PyQt 5.5 are installed at the same time), that's not always the case. For example, Ubuntu 16.10 comes with Qt 5.6 and PyQt 5.7.

So please use QT_VERSION instead of PYQT_VERSION here. We define QT_VERSION here:

https://github.com/spyder-ide/qtpy/blob/master/qtpy/__init__.py#L87